app/src/main/java/nerd/tuxmobil/fahrplan/congress/serialization/ScheduleChanges.kt

```diff
@@ -7,15 +7,18 @@ data class ScheduleChanges private constructor(
 
         val sessionsWithChangeFlags: List<SessionAppModel>,
         val oldCanceledSessions: List<SessionAppModel>,
-        val foundChanges: Boolean
+        val foundChanges: Boolean,
+        val foundAnyChanges: Boolean,
```

Comment:
This is quite confusing. If `foundChanges` wasn't about any change, it probably should have been named `foundUserRelevantChanges` or something similar.

Reply:
I tried to make it clear in the KDoc. Did you see these?

Reply:
:heavy_check_mark: Renamed now.